pkg/openstack/loadbalancer.go

```diff
@@ -430,6 +430,14 @@ func getSecurityGroupName(service *corev1.Service) string {
 	return securityGroupName
 }
 
+func getLoadBalancerIP(service *corev1.Service) string {
+	return service.Spec.LoadBalancerIP
```

Comment:
Should further annotation support be provided, enhancement can be realized here.

Reply:
enhanced in the commit https://github.com/kubernetes/cloud-provider-openstack/pull/2451/commits/58812808fa44205510af62d654c6e4e55c25bf9d.